ledger/sandbox/src/main/scala/com/digitalasset/platform/index/PostgresIndexer.scala

```diff
@@ -170,9 +173,25 @@ class PostgresIndexer private (
       case PartyAddedToParticipant(party, displayName, _, _) =>
         ledgerDao.storeParty(party, Some(displayName)).map(_ => ())(DEC)
 
-      case PublicPackagesUploaded(_, _, _, _) =>
-        // TODO (GS) implement once https://github.com/digital-asset/daml/pull/1610 lands
-        Future.successful(())
+      case PublicPackagesUploaded(archives, _, participantId, _) if archives.isEmpty =>
+        Future.successful {
+          PostgresIndexer.logger.warn(
```

Comment:
I don't think this needs to be logged at warn. INFO at most, but I'd also be happy with DEBUG.

Reply:
As discussed offline, since we agreed to switch from `List`s to individual packages, this piece of code will likely be removed in the following days, so I'll leave it as is.